packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -173,7 +173,7 @@ class SideNavItem extends ElementMixin(ThemableMixin(PolylitMixin(LitElement)))
   connectedCallback() {
     super.connectedCallback();
     this.__updateActive();
-    this.__boundUpdateActive = this.__updateActive.bind(this);
+    this.__boundUpdateActive = () => setTimeout(() => this.__updateActive());
```

Comment:
Let's move setting `this.__boundUpdateActive` to `constructor()` instead of `connectedCallback()`.
Also, binding is not necessary in this case as you changed the method to arrow function.